e2e.mk

```diff
@@ -0,0 +1,28 @@
+.PHONY: test-e2e-run
```

Comment:
One thing I thought of after looking at the conformance PR was that it might be beneficial to drop a comment in this file or where it's referenced in the Makefile explaining why these are split out. Just to keep us from getting a little too trigger-happy if we decide to do any refactoring around here.

Reply:
Fair.

Updated the other PR as well (which means it'll have to run many hours of E2E before we can merge it :))